components/brave_vpn/brave_vpn_service_desktop.cc

```diff
@@ -132,6 +145,7 @@ void BraveVpnServiceDesktop::ScheduleFetchRegionDataIfNeeded() {
 }
 
 void BraveVpnServiceDesktop::Shutdown() {
+  skus_service_.reset();
```

Comment:
seems weird here that we're calling reset, then calling the superclass shutdown then calling reset and clear on a bunch of other stuff. Should be cleaned up in a follow-up

Reply:
Fixed in https://github.com/brave/brave-core/pull/10847/commits/d634f38c925efbd98b7fe4cb274ae77d102b431d - moved to below where other resets are done